Project-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,53 @@
 import React from 'react';
 import './Display.css';
+import NumberButton from '../ButtonComponents/NumberButton';
+import ActionButton from '../ButtonComponents/ActionButton';
+import ClearButton from '../ButtonComponents/ClearButton';
+import Zero from '../ButtonComponents/Zero';
+import OutputDisplay from '../OutputDisplay';
+
+const CalculatorDisplay = props => {
+  return (
+    <div className="calculator">
+    <div className="buttons">
+      <div className="numButtons">
+        <div className="zero-btn">
+          <Zero />
+        </div>
```

Comment:
I think the overall structure of the buttons div needs to be reconfigured. I think you have the right idea with rows, but at the end of each row there should be an action. I wouldn't do the actions as a separate div. Incorporate it. You're making it too hard on yourself! You're so close! It'll be easier to style after you reconfigure it.